client/packages/host/src/api/hooks/help/useFeedbackForm.ts

```diff
@@ -27,37 +31,59 @@ export function useFeedbackForm() {
     setDraft(defaultDraft);
   };
 
+  const checkEmailIsValid = (email: string) => {
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    return emailRegex.test(email);
+  };
```

Comment:
Actually, could we add this function to [RegexUtils](https://github.com/msupply-foundation/open-msupply/blob/5737-validate-inputs-contact-form-frontend/client/packages/common/src/utils/regex/RegexUtils.tsx), and add some unit tests for it? 🙏 

Will be nice to have this as reusable when we implement more features involving emails

Reply:
Fixed in 039df99a1cd4844a2749c65e9760047125d7eea5 :)